modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita.adoc

```diff
@@ -300,7 +301,7 @@ Install the new license:
 
 . Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version.
```

Comment:
Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version. => Start the application server. Before you start Bonita Applications, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Bonita pages instead of the new version.

Reply:
I'm under the impression that this is for the migration before 7.7 so, since Portal is still there, it's why I left Portal for the section.

Reply:
This page contains a lot of stuff and is kind of hard to understand for me.

Reply:
ok